src/ApiService/ApiService/onefuzzlib/orm/Orm.cs

```diff
@@ -108,5 +109,20 @@ public async Task<TableClient> GetTableClient(string table, string? accountId =
             await tableClient.CreateTableIfNotExistsAsync(table);
             return tableClient.GetTableClient(table);
         }
+
+        public async Task<ResultOk<(int, string)>> Delete(T entity)
```

Comment:
should we rename this to ResultVoid (sometime later) ?

Reply:
yeah a simpler type might be useful even for the other methods